source/introduction.rst

```diff
@@ -33,10 +33,10 @@ stage concept. Platforms are free to adopt any other boot architecture.
    Example boot architecture
 
```

Comment:
The RFC says:
"Authors who follow these guidelines
   should incorporate this phrase near the beginning of their document:

      The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL
      NOT", "SHOULD", "SHOULD NOT", "RECOMMENDED",  "MAY", and
      "OPTIONAL" in this document are to be interpreted as described in
      [RFC 2119](https://datatracker.ietf.org/doc/html/rfc2119).
"

So I guess we should copy a lower-case version of that to the start of this spec.

Reply:
Added the statement to the start of the Intro Section.